nixos/modules/misc/nixpkgs.nix

```diff
@@ -33,7 +33,7 @@ let
   configType = mkOptionType {
     name = "nixpkgs-config";
     description = "nixpkgs config";
-    check = traceValIfNot isConfig;
+    check = x: if isConfig x then true else lib.traceSeqN 1 x false;
```

Comment:
I prefer the named one, actually.

Reply:
It’s locally `let`-named in the rebased commit.